src/components/LightUpChinatown/DonationRedirect.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import {
+  tabletScreens,
+  phoneScreens,
+} from '../../utilities/general/responsive';
+import ExternalLink from './ExternalLink.svg';
+
+const DonationRedirect = () => {
+  const { t } = useTranslation();
+
+  return (
+    <React.Fragment>
+      <Container>
+        <ColumnContainer>
+          <Title>{t('lightUpChinatown.goalReachedtitle')}</Title>
+          <br />
+          <br />
+          <Description>
+            {t('lightUpChinatown.goalReachedDescription1')}
+          </Description>
+          <br />
+          <Description>
+            {t('lightUpChinatown.goalReachedDescription2')}
+          </Description>
+        </ColumnContainer>
+        <ColumnContainer>
+          <Button
+            className={'button--outlined-with-red'}
+            onClick={(e) => {
+              e.preventDefault();
+              window.open(
+                'https://www.gofundme.com/f/light-up-chinatown',
+                '_blank',
+                'noopener,noreferrer'
+              );
+            }}
+          >
+            {t('lightUpChinatown.goalReachedButton1')}{' '}
+            <img alt="external-link" src={ExternalLink} />
+          </Button>
+          <Button
+            className={'button--outlined-with-red'}
+            onClick={(e) => {
+              e.preventDefault();
+              window.location.href =
+                'https://www.sendchinatownlove.com/ways-to-donate.html';
+            }}
+          >
+            {t('lightUpChinatown.goalReachedButton2')}
+          </Button>
+        </ColumnContainer>
+      </Container>
+    </React.Fragment>
+  );
+};
+
+export default DonationRedirect;
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  max-width: 1440px;
+  margin: auto;
+  @media (${tabletScreens}) {
+    flex-direction: column;
+    position: relative;
+  }
+`;
+
+const ColumnContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  @media (${tabletScreens}) {
+    padding: 5px;
+  }
+`;
+
+const Title = styled.div`
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  letter-spacing: 0.02em;
+  color: #ffffff;
+  @media (${phoneScreens}) {
+    font-size: 22px;
+  }
+`;
+
+const Description = styled.div`
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 22px;
+  letter-spacing: 0.02em;
+  color: #ffffff;
+  @media (${phoneScreens}) {
+    font-size: 13px;
+  }
+`;
+
+const Button = styled.div`
```

Comment:
Probably better to replace the `styled.div` with a `styled.a` and then styling it like a button and feeding it whatever attributes you need.

So rather than handling click through event handler, in application it'd look like:
```js
<Button href='https://www.gofundme.com/f/light-up-chinatown' rel='noopener noreferrer' target='_blank'/>
```

Reply:
Good call! I can try that out! I'll see if I can still have the gofundme open in a new tab, but keep our internal one as a redirect.